frame/support/procedural/src/construct_runtime/expand/call.rs

```diff
@@ -66,38 +67,6 @@ pub fn expand_outer_dispatch(
 				}
 			}
 		}
-		impl #scrate::dispatch::GetCallMetadata for Call {
-			fn get_call_metadata(&self) -> #scrate::dispatch::CallMetadata {
-				use #scrate::dispatch::GetCallName;
-				match self {
-					#(
-						#variant_patterns => {
-							let function_name = call.get_call_name();
-							let pallet_name = stringify!(#pallet_names);
-							#scrate::dispatch::CallMetadata { function_name, pallet_name }
-						}
-					)*
-				}
-			}
-
-			fn get_module_names() -> &'static [&'static str] {
-				&[#(
-					stringify!(#pallet_names),
-				)*]
-			}
-
-			fn get_call_names(module: &str) -> &'static [&'static str] {
-				use #scrate::dispatch::{Callable, GetCallName};
-				match module {
-					#(
-						stringify!(#pallet_names) =>
-							<<#pallet_names as Callable<#runtime>>::Call
-								as GetCallName>::get_call_names(),
-					)*
-					_ => unreachable!(),
-				}
-			}
```

Comment:
maybe those 2 functions should be kept, they can be used in the rust code sometime, it was asked by Gautam in the past if I recall correctly.